recipes/ffbidx/build.sh

```diff
@@ -0,0 +1,10 @@
+cuda_home=$(which nvcc)
+cuda_home=${cuda_home%/bin/nvcc}
+CUDA_HOME=${cuda_home} meson setup $MESON_ARGS --reconfigure -Ddefault_library=shared -Dinclude-python-api=enabled meson
```

Comment:
Looks like you neeed to add some args to help meson find NumPy in the host prefix.